_posts/2015-06-26-ux-part-2.md

```diff
@@ -0,0 +1,108 @@
+
+The OME UX team process is best illustrated by running through a real workflow that we have done over the last few releases of OMERO.
+
+##Sharing Rendering Settings
+
+The image below shows an image displayed in the full viewer of OMERO.insight version 5.0.5, with the rendering settings pane to the right. There are similar controls in the Preview pane of the Data Manager window in OMERO.insight and the same in OMERO.web. 
+
+Users can choose to view or hide channels, and use the sliders to adjust the minimum and maximum pixel intensities for each channel. Then when they are happy with the appearance of the image, they save their rendering settings to the server. They can do this for both their own images and save their own settings for other user’s images, if they are in a shared group.
+
+<img src="/images/use-cases-1.png" alt="" style="width: 600px;"/>
+
+*The full viewer and renderings setting pane in OMERO.insight version 5.0.5.*
+
+In version 5.0.5, a user's own settings were shown by default, and then to see settings saved by any other users, they had to click on the “Saved by” button at the top right.
+
+<img src="/images/use-cases-2.png" alt="" style="width: 400px;"/>
+
+*Button to view the settings saved by other users.*
+
+##User Requirements
+
+The usability of this part of the UI was less than satisfactory as:
+
+1. there is no way at all of seeing other user’s settings in the OMERO.web UI 
+2. we had been asked by users to show the image owner’s rendering settings by default, rather than the current user’s, and
+3. we had a few RFEs on the back burner for users to be able to save multiple rendering settings for each user.
+
+As it was thought we could make a bit of space in the schedule, the rendering settings sharing UI was added to the priorities list.
+
+Initially, a systematic survey of how the UIs currently handled setting and sharing of rendering settings was done and the results posted in a Google Drive document which everyone on the team has access to. 
+
+<img src="/images/use-cases-3.png" alt="Screenshot of user requirements for the rendering settings functionality." style="width: 600px;"/>
+
+*Documenting the user requirements.*
+
+Proposed changes to satisfy the user requirements were added.
+
+<img src="/images/use-cases-4.png" alt="Screenshot of possible solutions for the the user requirements." style="width: 600px;"/>
+
+*Suggestions for solutions.*
+
+What also needs to be taken into account is the fact that there are likely to be some server-side and other back-end changes needed to support any new or changed functionality in the UIs. The team uses Trello to keep track of works in planning and progress. The image below shows just some of the cards on our Trello boards that relate to the proposed rendering settings display changes.
```

Comment:
Drop a line that some boards are now public, since members of the community want to be more "involved" in the process (Paris 2015)